testapi.pm

```diff
@@ -224,6 +259,16 @@ sub assert_screen {
     return _check_or_assert($mustmatch, $timeout, 0);
 }
 
+=head2 check_screen
+
+  check_screen($mustmatch [,$timeout]);
+
+Wait for needle with tag C<$mustmatch> to appear on SUT screen.
+C<$mustmatch> can be string or C<ARRAYREF> of strings (C<['tag1', 'tag2']>).
+
+Returns matched needle or C<undef> if timeout is hit. Default timeout is 30s.
```

Comment:
I think also for a reader it is easier if the documentation is not "copy&paste and adapt" from assert_screen but referring to assert_screen, e.g. what do you think about:
```
Like C<assert_screen> but returns C<undef> if timeout is hit instead of throwing
an exception. Use this for optional matches.
```